src/dotnet-install.sh

```diff
@@ -755,6 +755,7 @@ download() {
         else
             unset http_code
             download_error_msg="Missing dependency: neither curl nor wget was found."
+            failed=true
```

Comment:
I would actually suggest exiting here. Any next actions, for example trying legacy link in this case doesn't make sense.

Reply:
Moreover it's almost 0 chance error - first step in the script checks if curl or wget is installed

Reply:
It is correct that this line of code won't execute because we check for the existence of wget and curl. But I still don't want to exit here in case we run this function from somewhere else.